Library/Homebrew/test/language/python_spec.rb

```diff
@@ -85,22 +85,6 @@
   let(:formula_bin) { dir/"formula_bin" }
   let(:formula) { double("formula", resource: resource, bin: formula_bin) }
 
-  describe "#create" do
-    it "creates a virtual environment" do
-      expect(formula).to receive(:resource).with("homebrew-virtualenv").and_return(resource)
-      subject.create
-    end
-
-    specify "virtual environment creation is idempotent" do
-      expect(formula).to receive(:resource).with("homebrew-virtualenv").and_return(resource)
-      subject.create
-      FileUtils.mkdir_p dir/"bin"
-      FileUtils.touch dir/"bin/python"
-      subject.create
-      FileUtils.rm dir/"bin/python"
-    end
-  end
```

Comment:
Would be good to still have some sort of tests here even if they don't use the resource.